data/sql/updates/pending_db_world/rev_1611776034016932100.sql

```diff
@@ -0,0 +1,51 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611776034016932100');
+
+DROP TABLE IF EXISTS `creature_template_resistance`;
+CREATE TABLE IF NOT EXISTS `creature_template_resistance` (
+  `CreatureID` MEDIUMINT(8) UNSIGNED NOT NULL,
+  `School` TINYINT(6) UNSIGNED NOT NULL,
+  `Resistance` SMALLINT(6) DEFAULT NULL,
+  `VerifiedBuild` SMALLINT(5) DEFAULT '0',
+  PRIMARY KEY (`CreatureID`, `School`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `creature_template_spell`;
+CREATE TABLE IF NOT EXISTS `creature_template_spell` (
+  `CreatureID` MEDIUMINT(8) unsigned NOT NULL,
+  `Index` TINYINT(6) UNSIGNED NOT NULL DEFAULT '0',
+  `Spell` MEDIUMINT(8) UNSIGNED DEFAULT NULL,
+  `VerifiedBuild` SMALLINT(5) DEFAULT '0',
```

Comment:
also here, why using string if it's an int ?

Reply:
I just copy-pasted from my Heidi. Gonna do it asap